### DISCUSSION TITLE
fatal error: gmock/gmock.h: No such file

### DISCUSSION THREAD

**Discussion by johndoe71rus** [diasurgical/DevilutionX#3949] [General]
Not sure if this is a issue.
Try build at lubuntu 20.04. 
Config without error https://pastebin.com/1n8fQkxf
Error at the end of build. https://pastebin.com/6zD0ypgY

 Am i something miss?

**Comment by AJenbo** [comment]
You probably didn't install `libgmock-dev`, see the latest instructions on https://github.com/diasurgical/devilutionX/blob/master/docs/building.md

**Comment by johndoe71rus** [comment]
yes. thanks
